core/src/message/mod.rs

```diff
@@ -230,3 +230,15 @@ pub trait Packet {
     /// Packet value.
     fn value(&self) -> Value;
 }
+
+/// The struct contains results of read only send message RPC call.
+#[derive(Clone, Debug, Decode, Encode, PartialEq, Eq, TypeInfo)]
+#[cfg_attr(feature = "std", derive(serde::Serialize, serde::Deserialize))]
+pub struct ReadOnlyReply {
```

Comment:
Why do we need it in core if you use it only in pallet. Even more - reimport it in pallet.

Reply:
Same as GasInfo. It's introduced in core to be shared between: pallets, runtime-apis and generated clients e.g. gsdk etc, where pallet will be excess